apps/antalmanac/src/components/Calendar/CalendarToolbar.tsx

```diff
@@ -346,6 +347,10 @@ function CalendarPaneToolbar(props: CalendarPaneToolbarProps) {
             <Box flexGrow={1} />
 
             <Box display="flex" flexWrap="wrap" gap={0.5}>
+                <Box display="flex" flexWrap="wrap" alignItems="center" gap={0.5}>
+                    <ScreenshotButton />
+                </Box>
+
                 <Box display="flex" alignItems="center" gap={0.5}>
                     <Tooltip title="Undo last action">
                         <IconButton onClick={handleUndo} size="medium" disabled={skeletonMode}>
```

Comment:
```suggestion
            <Box display="flex" flexWrap="wrap" alignItems="center" gap={0.5}>
                <ScreenshotButton />
                    
                <Tooltip title="Undo last action">
                    <IconButton onClick={handleUndo} size="medium" disabled={skeletonMode}>
```

issue: it's not entirely in the diff, but we've got a bit of unnecessary code duplication for styling. 

suggestion:  remove the Boxes wrapping the buttons around all the buttons and have them in a singular Box, like shown above? It's all styling `gap={0.5}` and `flexWrap="wrap` anyways

This, basically:

```tsx
            <Box display="flex" flexWrap="wrap" alignItems="center" gap={0.5}>
                <ScreenshotButton />

                <Tooltip title="Undo last action">
                    <IconButton onClick={handleUndo} size="medium" disabled={skeletonMode}>
                        <UndoIcon fontSize="small" />
                    </IconButton>
                </Tooltip>

                <Tooltip title="Clear schedule">
                    <IconButton onClick={handleClearSchedule} size="medium" disabled={skeletonMode}>
                        <DeleteIcon fontSize="small" />
                    </IconButton>
                </Tooltip>

                <CustomEventDialog key="custom" scheduleNames={AppStore.getScheduleNames()} />
            </Box>
```